src/test/paths.ts

```diff
@@ -55,6 +55,15 @@ describe('paths', () => {
                     }),
                     'file:///app/source.php'
                 )
+                // longest prefix matching for server paths
+                assert.strictEqual(
+                    convertClientPathToDebugger('/home/felix/mysource/subdir/source.php', {
+                        '/var/www': '/home/felix/mysite',
+                        '/app': '/home/felix/mysource',
+                        '/app/subdir1': '/home/felix/mysource/subdir',
```

Comment:
The list should be built so that without the patch the test fails. Please move `'/app/subdir1': '/home/felix/mysource/subdir',` one line up, before `'/app': '/home...`.

Reply:
Good catch!